### PR TITLE
:truck: update: 2nd talk details

### PR DESCRIPTION
**Pull request by tusharnankani**
Will merge after: 
- Receiving and updating the Slide Deck
- Merging https://github.com/OurTechCommunity/catchup/pull/138

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -193,23 +169,38 @@ <h3>OTC Talks #2</h3>
...
+						<a
+							href="https://twitter.com/omtalk"
+							target="_blank"
+							rel="noreferrer"
+							>Omkar Khair</a
+						>
+					</p>
+	
+					<p>
+						<a href="/2.html">More details</a>
```
Comment (HarshKapadia2) [inline index.html]: Please remember to change this from `/2.html` to `/2`.

Comment (tusharnankani) [reply index.html]: Yes.

**Comment by tusharnankani** [comment]
Just need the slide deck now.

**Comment by tusharnankani** [comment]
Done. Merging!

**Comment by tusharnankani** [comment]
image

**Pull request merged**